packages/site/tsconfig.base.json

```diff
@@ -5,7 +5,7 @@
         "module": "ESNext",
         "moduleResolution": "Node",
         "noImplicitAny": true,
-        "jsx": "react",
+        "jsx": "react-jsx",
```

Comment:
This change was need due to errors showing up for missing react for jsx context content

https://stackoverflow.com/questions/64656055/react-refers-to-a-umd-global-but-the-current-file-is-a-module